### ISSUE TITLE
glfwGetWindowPosition() incorrect when using multiple monitors

### ISSUE THREAD

**Issue by Whirvis** [glfw/glfw#1532]
When attempting to retrieve the window position via the ``glfwGetWindowPosition()`` method or the callback for when the window is moved, the position will start to report incorrect positioning when using multiple monitors.

Graphics card: NVidia GTX 1080
Operating System: Windows 10
Programming language: Java
Linking library: LWJGL 3
Monitors connected: 4
Monitor order (according to Windows): 1, 3, 4, 2
Monitor information:
 - Monitor 1: 3840x2160
 - Monitor 2: 1024x768
 - Monitor 3: 3840x2160
 - Monitor 4: 1920x1080

[Video showing the issue](https://github.com/glfw/glfw/files/3388053/issue_video.zip).
In the video, the top left is monitor 1 (set to the most left), the top right is monitor 3 (ordered to the center), the bottom right is monitor 4 (ordered to the most right and top), and the bottom left is the game and console output. Monitor 2 is not shown in the video.

The code to replicate this is simply by using either the GLFW window position callback or ``glfwGetWindowPosition()``.

**Comment by Whirvis** [comment]
My first guess as to what causes this issue might be due to GLFW not accounting for order monitoring not being sequential? I'll try making my orders go from 1 to 4 in sequence to see if that fixes the issue.

**Comment by Whirvis** [comment]
It seems that when a window goes onto a monitor other than the monitor it was created on, it adds on to the X position of the window by the width of the new monitor the window is on. Subtracting by the width of the original monitor and the new monitor seems to actually return the X position of the window.

**Comment by Whirvis** [comment]
After doing some more testing, it appears that this bug triggers whenever more than half the window is moved over into the other monitor. Before this point, the X position is accurate. If the window is stretched big enough, it can be moved all the way over to the leftmost monitor and the X position accurately return -3840.

**Comment by elmindreda** [comment]
Check the DPI Awareness column in the Details tab of Task Manager and see what awareness level your program has.

**Comment by Whirvis** [comment]
For whatever reason, I have no ability to enable a DPI awareness column in the details tab of the task manager. The option isn't there.

**Comment by elmindreda** [comment]
Your version of Windows 10 may be older than build 18262.

Do all your monitors use the same scaling factor? Also, does your program do anything else before initializing GLFW?

**Comment by Whirvis** [comment]
No, not all of my monitors use the same scaling factor. Monitor 1 and 2 use 150%, and monitors 2 and 4 use 100%. I'll see what changing them to all 150% does. 

The program does not do anything special before initializing GLFW. It does some stuff relating to the engine (such as creating needed files, loading configuartions, etc.) but nothing related to GLFW or other graphics beforehand.
